packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswer/templates/generatedAnswer.html

```diff
@@ -7,7 +7,7 @@
           >{labels.generatedAnswerForYou}</span
         >
         <div class="generated-answer__header-actions slds-grid">
-          <template lwc:if={shouldDisplayActions}>
+          <template lwc:if={shouldDisplayFeedback}>
```

Comment:
This was named, `shouldDisplayActions` cause it is a boolean that determines if the feedback and the copy to clipboard button, so it's not  not only about the feedback

Reply:
ah oops, didn't know about the copy to clipboard, it was also used to decide if we wanted to display the rephrase